model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -327,6 +335,13 @@ message ProcessBundleResponse {
   // payload field with the bytes specified here.
   map<string, bytes> monitoring_data = 5;
 
+  // (optional) Output elements of the processed bundle. Either all or
+  // none of the bundle elements should be included in the ProcessBundleResponse.
+  // This elements embedding is to achieve better Fn API efficiency for bundles
+  // that only produce small amount of outputs. See more at
+  // https://s.apache.org/beam-fn-api-control-data-embedding.
```

Comment:
```suggestion
  // (optional) Output elements of the processed bundle. Either all or
  // none of the bundle elements should be included in the ProcessBundleResponse.
  // This embedding is to achieve better efficiency for bundles
  // that contain only small amounts of data. See more at
  // https://s.apache.org/beam-fn-api-control-data-embedding.
```

Reply:
Please add another comment stating that: `This field can only be set if the runner declares that they support the "...." capability.`

Reply:
Done.

Reply:
What do you think of my first comment which updated the existing text as well?